tools/setupResourceScripts/setup_tdr_resources.py

```diff
@@ -389,6 +400,15 @@ def main():
                 )
                 print("Added snapshot builder settings for snapshot %s" % snapshot_id)
 
+    print("Creating snapshot access requests on snapshots")
+    snapshot_access_request_ids = []
+    for snapshot_id in snapshot_ids:
+        snapshot_access_request_id = create_snapshot_request(clients,
+                                                         snapshot_id)
+        snapshot_access_request_ids.append(snapshot_access_request_id)
+        print(
+            f"Created snapshot access request {snapshot_access_request_id} on snapshot {snapshot_id}")
```

Comment:
If I'm understanding this correctly, this would automatically create a snapshot request on any snapshot created with this script.
If that's right - I don't think we want to do that. We want this script to be more general purpose so that anyone can create dataset and snapshots. I think we would also expect for this to fail for datasets that do not have snapshot builder settings set. 
I think it would be good to make this an option on the suite file - This is probably getting a bit out of scope, so I'm good with moving this work to another ticket, but I would want to remove or comment out this chunk of code for now.